app/components/notification-list-row/template.hbs

```diff
@@ -0,0 +1,19 @@
+<div class="flex-container">
+    {{#if updateFrequency.isRunning}}
+        {{loading-indicator classes='ball-dark text-center'}}
```

Comment:
Would you mind using a skeleton screen instead of a the ball scale loading here as well? We are in the process of phasing those loaders out.